lua/lualine/component.lua

```diff
@@ -76,7 +76,9 @@ function M:apply_padding()
       -- When component has changed the highlight at begining
       -- we will add the padding after the highlight
       local pre_highlight = vim.fn.matchlist(self.status, [[\(%#.\{-\}#\)]])[2]
-      self.status = pre_highlight .. string.rep(' ', l_padding) .. self.status:sub(#pre_highlight + 1, #self.status)
+      self.status = pre_highlight
```

Comment:
Did you change anything here other than formating ?

Reply:
I Don't remember that line, so it probably got auto formatted and I didnt notice. My bad.

Reply:
I reverted the change. My neovim should utilizes project specific `.stylua.toml` settings so I'm surprised that occurred.